src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -360,8 +348,16 @@
       <Output TaskParameter="ConsoleOutput" PropertyName="VersionHostName" />
     </Exec>
 
-    <PropertyGroup>
-      <BuiltByString Condition="'$(BuiltByString)' == '' AND '$(VersionUserName)' != '' AND '$(VersionHostName)' != ''">%20built by: $(VersionUserName)-$(VersionHostName)</BuiltByString>
+    <!-- Set BuiltByString (which is added to the end of the informational version) -->
+    <!-- This string has additional information as well (like the GIT commit URL) -->
+    <!-- and has the general form @Name1: Value1 @Name2: Value2  ... which makes it easy to parse -->
+    <PropertyGroup Condition="'$(BuiltByString)' == ''">
+      <!-- Set the @BuiltBy key-value pair -->
+      <BuiltByString Condition="'$(VersionUserName)' != '' AND '$(VersionHostName)' != ''">$(BuiltByString)%20%40BuiltBy: $(VersionUserName)-$(VersionHostName)</BuiltByString>
```

Comment:
My guess is that it would not matter either way, I was simply avoiding changing anything that I did not need to change.  The original text I was fiddling with had a %20built by: prefix (that is they used %20 for the space), so I did the same.  

Given that there are other spaces in the string, it will almost certainly work with a leading space, so if you want, I can change it (below as well).  Let me know.  

Note that I used %40 for the @ out of an abundance of caution.   Since these strings get used in MSBuild statements and @ is a special character for MSBuild it is likely to cause grief.     We could pick something else, I just want something that is 'letter like' (it has been used in identifiers for some language or another), but is very unlikely to ever be needed in the 'value' part of the key-value pairs (so we don't have the complexity of escape sequences).